.github/workflows/wai-trigger-pr.yml

```diff
@@ -1,7 +1,9 @@
-name: Trigger pull request creation for wai repository
+name: Trigger PR update for WAI-APG site
 
 on:
   pull_request:
+    branches-ignore:
+      - "dependabot/**"
```

Comment:
Well perhaps, but my feeling on it was that `w3c/wai-aria-practices` doesn't directly depend on the dependencies coming from this repository. `w3c/wai-aria-practices` is consuming `aria-practices.html`, so if `w3c/aria-practices` dependencies prevented `aria-practices.html` from being properly built then it would be representative of another problem to be resolved here.

On the flip side, I don't have a strong feeling against not doing this and can't see a problem with it so I can make the change. But the trigger will now need to be `pull_request_target` I believe for dependabot to access the token?

Reply:
Done @nschonni

Reply:
You're right, it's more the wai-trigger-cleanup.yml that has the script in the repo, but a PR won't actually exercise it anyway, so I think you can ignore this comment. Not sure if the `paths` need to be trimmed down though. Probably the `"package*.json"` was the original thing my brain saw

Reply:
> You're right, it's more the wai-trigger-cleanup.yml that has the script in the repo, but a PR won't actually exercise it anyway, so I think you can ignore this comment. 

Alright, I can add it back.

> Not sure if the `paths` need to be trimmed down though. Probably the `"package*.json"` was the original thing my brain saw

Removing `"package*.json"` is fine with me taking another a look but I suppose we should also now be adding `"index.html"` as well 😅. I don't believe I have full context on some of the paths so I tried take into account all the `pull_request` trigger related paths. Is there any other that may stand out to you?

Reply:
Sorry, don't let me derail this 😄 
I don't think I followed what these workflows were doing, so you've probably got it right.

Reply:
Ha, no problem. Perhaps this can help to clear up what's happening? https://github.com/w3c/wai-aria-practices/issues/13

Reply:
Merging this in @nschonni but we can continue the discussion in a separate issue if anything else pops out to you